packages/recommend-react/src/__tests__/useTrendingItems.test.tsx

```diff
@@ -43,6 +43,7 @@ describe('useTrendingItems', () => {
         },
         facetName: 'test4',
         facetValue: 'test3',
+        itemComponent: (item) => <>{item}</>,
```

Comment:
is this change required? maybe it should be done separately from this PR

Reply:
I can do it in another PR that gets merged before 👍

Reply:
This was discussed with @francoischalifour as we don't want to export internal details of how the context works to the end users, while keeping them accessible internally.

I don't think file structure should change here.

Reply:
✅ :https://github.com/algolia/recommend/pull/131/commits/2af99ace34588ed75b07db4bc3ef619872a9751e